City-of-Helsinki/kukkuu-ui
# KK-672 | Add logo of Kaapeli to the home page

## Description

Adds logo of Kaapeli to the home page.

## Context




[KK-672](https://helsinkisolutionoffice.atlassian.net/browse/KK-672)

## How Has This Been Tested?

I've updated the snapshot tests to reflect the addition of the logo.

## Manual Testing Instructions for Reviewers

1. Access home page
1. Scroll down until you see the logos
1. Expect to see "Kaapeli" logo
1. Expect it to be in correct alphabetical order
